app/features/analysis/edit_set/OneRMEditSetView.js

```diff
@@ -37,8 +39,17 @@ class OneRMEditSetView extends Component {
     // RENDER
 
     _renderNavigation() {
-        if (Platform.OS === 'ios') {
+        if (Platform.OS === 'ios' && !Device.isiPhoneX()) {
             var statusBar = (<View style={{height: 20, width: 9001, backgroundColor: 'black'}}></View>);
+        } else if (Device.isiPhoneX()) {
```

Comment:
- [ ] make it if device is iphonex else if platform == ios to simplify logic so you don't do that extra check